runtime_typecheck/runtime_typecheck.py

```diff
@@ -147,10 +147,10 @@ def __repr__(self) -> str:
         if self.name is None:
             return f'Generic type error: {self.generic_message}'
         if self.missing_parameter:
-            return f'{self.name} has no default value and was not given,expected a value of type {self.expected_type}'
-        else:
```

Comment:
`else` not needed because the previous `if` would either `return` or continue into the `else` branch regardless.